lib/ui/widgets/pokeball_background.dart

```diff
@@ -24,7 +24,7 @@ class PokeballBackground extends StatelessWidget {
     final iconSize = IconTheme.of(context).size;
```

Comment:
```suggestion
    final iconSize = IconTheme.of(context).size ?? 0;
```
And then you can remove the `!` below